format/spec.md

```diff
@@ -200,12 +200,16 @@ Supported primitive types are defined in the table below. Primitive types added
 |                  | **`uuid`**         | Universally unique identifiers                                           | Should use 16-byte fixed                         |
 |                  | **`fixed(L)`**     | Fixed-length byte array of length L                                      |                                                  |
 |                  | **`binary`**       | Arbitrary-length byte array                                              |                                                  |
+| [v3](#version-3) | **`geometry(C, CE, E)`** | An object of the simple feature geometry model as defined by Appendix G; This may be any of the geometry subclasses defined therein; crs C [4], crs-encoding CE [5], edges E [6] | C, CE, E are fixed, and if unset will take default values. |
```

Comment:
I think maybe we should just link out for the requirements here since it's a bit complicated.

The description as well could be 

Simple feature geometry [Appendix G](#appendix-g), Parameterized by ....

I also don't think we should allow it to be unset ... can we just require that a subclass is always picked?